src/test/resources/com/twingly/search/client/tapes/search_without_valid_API_key.yaml

```diff
@@ -0,0 +1,24 @@
+!tape
+name: search without valid API key
+interactions:
+- recorded: 2016-03-12T08:56:47.276Z
+  request:
+    method: GET
+    uri: https://api.twingly.com/analytics/Analytics.ashx?key=API_KEY&xmloutputversion=2&searchpattern=some+pattern
+    headers:
+      Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2
+      Connection: keep-alive
+      Content-Length: '0'
+      Host: api.twingly.com
+      User-Agent: Twingly Search Java Client/null
```

Comment:
Happened to notice `/null` here, is there some problem with reading the version from the property file?